src/pages/auth/authSlice.ts

```diff
@@ -0,0 +1,37 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RootState } from '../../store';
+
+import { CounterState, User } from '../../types';
+
+const initialState = {
+  isAuthenticated: false,
+  isInitialized: false,
+  user: null
+} as CounterState;
```

Comment:
```suggestion
const initialState: CounterState = {
  isAuthenticated: false,
  isInitialized: false,
  user: null
};
```

Tránh dùng `as` trừ trường hợp bất khả kháng. Tương tự với `any`